### ISSUE TITLE
Can not log in wechat

### ISSUE THREAD

**Issue by pfannenwenderer** [meetfranz/franz#643]
When I scan the QR code, I get this message:
<img width="1440" alt="bildschirmfoto 2018-02-02 um 10 09 45" src="https://user-images.githubusercontent.com/36069329/35725359-6a878ba8-0801-11e8-8aa6-0b27675cdbf2.png">

It is telling me to use the app for mac or windows and not use the browser. WeChat for Mac works fine for me. WeChat in my browser does not work. Now I want to use it in meetfranz and it does not work :(

please help

**Comment by pfannenwenderer** [comment]
Please help :(

Zur Gewährleistung der Sicherheit Ihres Kontos wurde die Anmeldung bei WeChat über das Internet ausgesetzt. Melden Sie sich stattdessen auf einem Computer bei WeChat für Windows oder WeChat für Mac an. Laden Sie WeChat für Windows oder Mac herunter von http://wechat.com.

**Comment by haraldox** [comment]
@pfannenwenderer could not reproduce, do you still have this issue?

**Comment by pfannenwenderer** [comment]
Yes, I still get the same error :( But I also get this error when I use chrome or firefox.

**Comment by pfannenwenderer** [comment]
I will also ask the wechat support now :)

**Comment by BennSenn** [comment]
@pfannenwenderer Same problem here. Any feedback from the WeChat support yet?

**Comment by pfannenwenderer** [comment]
Not yet :( 

Maybe after the chinese holiday

Von meinem iPhone gesendet

**Comment by koma-private** [comment]
FYI, I also posted same message to #service on Franz's Slack as follow:
> I found a thread on this message (https://www.v2ex.com/t/435567)
They are suggesting, this message will be shown if an user account is too new or considered as a robot (by mistake.)

**Comment by Raphael-C** [comment]
Same here !

Anyone found a fix ?

For account security, newly registered WeChat accounts are unable to log in to Web WeChat. To use WeChat on a computer, use Windows WeChat or Mac WeChat at http://wechat.com

**Comment by perhl** [comment]
Same to me!
"Zur Gewährleistung der Sicherheit Ihres Kontos wurde die Anmeldung bei WeChat über das Internet ausgesetzt. [...]"

**Comment by bvrulez** [comment]
Same problem here. Maybe it is related to not giving private contacts free in the app. And maybe because I just have one contact using WeChat.

**Comment by BerndErnst** [comment]
How is the status? @koma-private

**Comment by koma-private** [comment]
@BerndErnst 
If you experience problems when using WeChat via Franz, please follow these steps.

Step 1) Please try to login to your account by browsing their official web interface, [Web WeChat]( https://web.wechat.com)
If you experience same problem even using Web WeChat, then proceed to Step 2.
If you can use Web WeChat without any problem, then proceed to Step 3.

Step 2) The problem may be caused by WeChat service itself. Ask their support at [WeChat Help Center](https://help.wechat.com/)

Step 3) 
Something wrong with WeChat recipe, Franz or Electron. Please report here with detail information.

**Comment by BerndErnst** [comment]
@koma-private 
When I try to login I get the QR code. When I scan it I get this message in Franz:

<error><ret>1203</ret><message>For account security, newly registered WeChat accounts are unable to log in to Web WeChat. To use WeChat on a computer, use Windows WeChat or Mac WeChat at http://wechat.com</message></error>

**Comment by koma-private** [comment]
@BerndErnst 
Thank you for reporting, did you also try to login to Web WeChat? https://web.wechat.com/

**Comment by BerndErnst** [comment]
This is from https://web.wechat.com/
I want to add that my WeChat Acc is actually at least 1 year old.
And don't say it's a clear case in which you can't do anything without becoming too hacky ;)

<error><ret>1203</ret><message>For account security, newly registered WeChat accounts are unable to log in to Web WeChat. To use WeChat on a computer, use Windows WeChat or Mac WeChat at http://wechat.com</message></error>

![wechatweb](https://user-images.githubusercontent.com/9050259/41843619-41f05ade-786e-11e8-975e-603247efd292.JPG)

**Comment by koma-private** [comment]
@BerndErnst 
Hi. Maybe this is a bad news. I found a post at other project on GitHub:

> This issue is related to the latest policy changing from Tencent: Newly created Wechat Account will not be able to login via Web Wechat anymore. 
> The solution will be: use a Wechat Account that registered before Jun 2017.

And it seems Tencent, the developer of WeChat has changed their security policy to ban chat-bots working on Web WeChat. Although there is little hope, you can ask their support at [WeChat Help Center](https://help.wechat.com/)

**Comment by LordNacho** [comment]
Hi, not sure if this is useful, but this other project claims to have found a solution:

https://github.com/Chatie/wechaty/issues/872

**Comment by jchacana** [comment]
Have you guys found a solution for this?? I'd really like to use wechat with franz

**Comment by kvdv99** [comment]
Same here !

..."For account security, newly registered WeChat accounts are unable to log in to Web WeChat. To use WeChat on a computer, use Windows WeChat or Mac WeChat at http://wechat.com"..

Anyone found a fix ?

**Comment by LordNacho** [comment]
Since my last post, I tried it with someone who had an account from before the cutoff, and it works just fine.

**Comment by qwe-hue** [comment]
lucky u
i can not

**Comment by T-use** [comment]
I have a WeChat account since around 2010, and I cannot use the web browser version either. The "new accounts since 2017" official response seems like a bit of a cop out.

**Comment by feidaman** [comment]
WeChat is a pile of cr*p. I used to prefer the in-browser version because I was able to zoom in to make Chinese characters larger and to use my browser's Chinese popup translator extension, neither of which features exists in the desktop version, but now the web version no longer works so...

**Comment by enTerbury** [comment]
same problem, also using an account from well before 2017.

honestly, it's most probably just an excuse to let foreigners download the installation version. spyware/back door? not unlikely.
